src/Sylius/Bundle/AttributeBundle/DependencyInjection/SyliusAttributeExtension.php

```diff
@@ -56,6 +56,7 @@ public function load(array $config, ContainerBuilder $container)
 
                 if (false !== strpos($resourceName, 'value')) {
```

Comment:
I just add a [line](https://github.com/xantrix/Sylius/blob/2d22b012accdc5a98aa825ace10cffd47a5f5f81/src/Sylius/Bundle/AttributeBundle/DependencyInjection/SyliusAttributeExtension.php#L59) here to inject my service in [BuildAttributeValueFormSubscriber](https://github.com/xantrix/Sylius/blob/2d22b012accdc5a98aa825ace10cffd47a5f5f81/src/Sylius/Bundle/AttributeBundle/Form/Type/AttributeValueType.php#L74)